CONTRIBUTING.md

```diff
@@ -1,54 +1,97 @@
-# Getting Started
-Install Gourmet's dependencies:
-
-    pip3 install -r requirements.txt
-    pip3 install -r development.txt
-
-`requirements.txt` contains the dependencies needed for Gourmet itself,
-`development.txt` contains the packages required for testing.
-
-Before you start development, you should first build localized *.mo and
-*.gourmet-plugin files within a build/ subdirectory of the source tree by
-running
-
-    python3 setup.py build_i18n
-
-You can then install Gourmet in edit mode so:
+# Contributing
 
-    pip3 install --user -e .
+Thank you for taking an interest in contributing to Gourmet! We appreciate that
+you're thinking about offering your time to improving the project, and it's our
+goal to respect your contribution accordingly.
 
-By doing so, you will be able to test your changes when launching Gourmet.
+Although this document focuses on code contributions, you can contribute in
+several ways:
+- File a bug report.
+- Add or improve documentation.
+- Promote the project to others.
 
-# Style
-New code follows the [PEP 8](http://www.python.org/dev/peps/pep-0008/) standard.  
 
-The following conventions are not strictly followed in Gourmet, old code
-should be reformatted only when modifying it.
+## Contributing Code
 
-# Getting the Source
-Gourmet's source code is hosted on [GitHub](https://github.com/thinkle/gourmet).
-You can clone it by opening a command prompt and typing:
+In general, the process for contributing code is:
 
-    git clone https://github.com/thinkle/gourmet.git
+1. Pick or open an [issue](https://github.com/thinkle/gourmet/issues) to work
+   on
+2. Post a comment expressing your intent to make sure nobody else is already
+   working on it
+3. Set up a development environment, as described below
+4. Hack the code, and when ready
+5. Push your changes to your forked repo and create a pull request.
 
-If you don't have any previous experience with Git, you might want to take a
-look at the [official Git tutorial](http://www.kernel.org/pub/software/scm/git/docs/gittutorial.html),
-or the [GitHub Help pages](http://help.github.com/).
+Make sure to also check out our upcoming [milestones](https://github.com/thinkle/gourmet/milestones).
 
-# Contributing
-## Development
-If you'd like to contribute,
 
-1. [fork](https://github.com/thinkle/gourmet/fork) and clone the repository
-2. pick an [issue](https://github.com/thinkle/gourmet/issues) to work on
-3. post a little comment expressing your intent to make sure nobody else is
-   already working on it
-4. hack the code, and when ready
-5. push to your forked repo, and create a pull request.
+## Setting Up a Development Environment
+
+You'll want to clone Gourmet to your computer and probably
+[fork](https://github.com/thinkle/gourmet/fork) it as well.
+
+Ensure your system has the necessary prerequisites installed:
+- [Python](https://www.python.org/), which is what Gourmet is written in. Only
+  currently-supported versions of Python 3 are supported.
```

Comment:
I've updated the documentation, but since Debian-based distros pin their package versions with every release, this really shouldn't be an issue. Assuming Gourmet gets included in the official Ubuntu repositories, it will be whatever version is current at the time the release is frozen, which will presumably correspond with a version of Python that is supported at the time. IMO, the question is more: do you want to support old versions of Gourmet that may be in current (LTS) versions of Ubuntu (or Debian, or Mint, etc.)?